Source/Charts/Renderers/YAxisRendererHorizontalBarChart.swift

```diff
@@ -253,41 +253,44 @@ open class YAxisRendererHorizontalBarChart: YAxisRenderer
             // if drawing the limit-value label is enabled
             if l.drawLabelEnabled, !label.isEmpty
             {
-                let labelLineHeight = l.valueFont.lineHeight
-                
+                let labelLineSize = label.size(withAttributes: [.font: l.valueFont])
+                let labelLineRotatedSize = labelLineSize.rotatedBy(degrees: l.labelRotationAngle)
+                let labelLineRotatedWidth = labelLineRotatedSize.width
+                let labelLineRotatedHeight = labelLineRotatedSize.height
+
                 let xOffset = l.lineWidth + l.xOffset
                 let yOffset = 2.0 + l.yOffset
+                let labelRotationAngleRadians = l.labelRotationAngle.DEG2RAD
 
-                let align: TextAlignment
                 let point: CGPoint
+                let anchor = CGPoint(x: 0.0, y: 0.0)
 
                 switch l.labelPosition
                 {
                 case .rightTop:
-                    align = .left
                     point = CGPoint(x: position.x + xOffset,
                                     y: viewPortHandler.contentTop + yOffset)
 
                 case .rightBottom:
-                    align = .left
                     point = CGPoint(x: position.x + xOffset,
-                                    y: viewPortHandler.contentBottom - labelLineHeight - yOffset)
+                                    y: viewPortHandler.contentBottom - labelLineRotatedHeight - yOffset)
 
                 case .leftTop:
-                    align = .right
-                    point = CGPoint(x: position.x - xOffset,
+                    point = CGPoint(x: position.x - xOffset - labelLineRotatedWidth,
```

Comment:
let's put `labelLineRotated_Width/Height` at the same position? either at the last or in middle